cypress/fixtures/videos.json

```diff
@@ -0,0 +1 @@
+ {"videos":[{"id":327,"movie_id":632618,"key":"u2ncERi6TgU","site":"YouTube","type":"Trailer"}]}
```

Comment:
I wonder what the purpose of `"type":"Trailer"` is. I know this is just how that API is set up and doesn't affect our code at all. I'm just curious about it.